js/ed11y-element-panel.js

```diff
@@ -98,6 +105,9 @@ class Ed11yElementPanel extends HTMLElement {
     // Open the button
     goto.setAttribute('data-ed11y-action','open');
 
+    // Set counter
+    Ed11y.issueCounter.innerHTML = 'Issue ' + (goNum + 1) + '/' + Ed11y.totalCount;
```

Comment:
Add this via a new `en.panelCurrentIssue` function in `ed11y-localization.js`, similar to the `en.buttonShowHiddenAlerts` function you'll find toward the top of that file now. Then call the function here with `Ed11y.M.panelCurrentIssue`.